source/uwp/Renderer/lib/Util.cpp

```diff
@@ -745,6 +745,243 @@ HRESULT GetBackgroundColorFromStyle(ABI::AdaptiveNamespace::ContainerStyle style
 }
 CATCH_RETURN;
 
+HRESULT GetFontDataFromStyle(
+    _In_ ABI::AdaptiveNamespace::IAdaptiveHostConfig* hostConfig,
+    _In_ ABI::AdaptiveNamespace::FontStyle style,
+    _In_ ABI::AdaptiveNamespace::TextSize desiredSize,
+    _In_ ABI::AdaptiveNamespace::TextWeight desiredWeight,
+    _Out_ HSTRING* resultFontFamilyName,
+    _Out_ UINT32* resultSize,
+    _Out_ ABI::Windows::UI::Text::FontWeight* resultWeight) noexcept try 
+{
+    RETURN_IF_FAILED(GetFontFamilyFromStyle(hostConfig, style, resultFontFamilyName));
+    RETURN_IF_FAILED(GetFontSizeFromStyle(hostConfig, style, desiredSize, resultSize));
+    RETURN_IF_FAILED(GetFontWeightFromStyle(hostConfig, style, desiredWeight, resultWeight));
+    return S_OK;
+} CATCH_RETURN;
```

Comment:
I'm not sure we're getting much value out of this function :)

Reply:
So, this is the frustrating part I was referring to under "Notes". This is handling the inheritance for the FontFamily, FontSizes, and FontWeights. It's already implemented in the ObjectModel's HostConfig but, in order to access that, you need an instance of the HostConfig. Specifically, you would need an instance every time you need to access the FontFamily, FontSize, or FontWeight value.